lightning/src/ln/channelmanager.rs

```diff
@@ -378,7 +394,8 @@ impl HTLCSource {
 	}
 }
 
-struct InboundOnionErr {
+/// Invalid inbound onion payment.
+pub struct InboundOnionErr {
 	err_code: u16,
 	err_data: Vec<u8>,
 	msg: &'static str,
```

Comment:
Oops, these fields should be made `pub`.

Reply:
ok, done